sqe-dto/Transformation.cs

```diff
@@ -4,8 +4,11 @@ namespace SQE.API.DTO
 {
     public class TransformationDTO
     {
-        public float? scale { get; set; }
-        public float? rotate { get; set; }
+        [Range(0, 99.9999)]
+        public float scale { get; set; }
```

Comment:
Does someone enforce the Range? Cool! Anyway, I would make the Range for Scale run from 0.01 to 100, which means we can make it 100 times smaller or 100 times larger.

Reply:
Yeah, dotnet does the enforcement when it marshalls the HTTP request JSON body into a c# object.  It sends a 400 error when these constraints aren't met.  The number ranges just come straight from the underlying SQL data type: scale is a decimal(6,4), so two digits before the decimal point and 4 after (3 bytes total storage space), and rotate is decimal(6,2), 4 digits before the decimal and two digits after (also 3 bytes).  I had to use decimal in the DB due to some problems with floating point equality comparisons and the whole database writing system.  I can set these however you want, and using 0.1 instead of 0 is a great idea (sadly, I can't give you 100 though).  

What should I do to help you know that floating point values with many digits after the decimal will be truncated?  I considered precision to the millionth sufficient for scale and to the hundredth sufficient for degrees of rotation (I doubt we even have real precision in the SVG browser representation at these levels).